src/charm.py

```diff
@@ -257,6 +257,7 @@ def _login_ui_layer(self) -> Layer:
                 "PORT": str(APPLICATION_PORT),
                 "BASE_URL": self._domain_url,
                 "TRACING_ENABLED": False,
+                "AUTHORIZATION_ENABLED": False,
```

Comment:
are we sure about this?

Reply:
There is no `openfga` relation anyways, so it can't be enabled until we add it